include/HUDL/HUDL.hpp

```diff
@@ -1,6 +1,11 @@
 #ifndef _HUDL_
 #define _HUDL_
 
+#include "EVT/dev/LCD.hpp"
```

Comment:
Change to `<>` includes

Reply:
Kind of weird to use <> for non standard library headers but sure?